docs/content/ingestion/firehose.md

```diff
@@ -27,6 +27,31 @@ This firehose acts as a Kafka consumer and ingests data from Kafka.
 
 This firehose ingests events from a predefined list of S3 objects.
 
+#### StaticAzureBlobStoreFirehose
```

Comment:
On second look, it seems that the other ones do not mention anything special (I'm looking at YOU sattic s3 firehose!) I would just mention the data is newline delimited, and parsed as per the `InputRowParser` configuration